cms/envs/common.py

```diff
@@ -2686,3 +2686,5 @@
 COURSE_LIVE_GLOBAL_CREDENTIALS = {}
 
 PERSONALIZED_RECOMMENDATION_COOKIE_NAME = 'edx-user-personalized-recommendation'
+
+ENROLLMENT_CONFIRMATION_EMAIL_REDESIGN = True
```

Comment:
Shouldn't we default to False and enable it through remote config?

Reply:
Agreed